h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -300,6 +300,22 @@ public static double l2norm(double [] x, boolean skipLast){
     for(int i = 0; i < a.length; i++ ) a[i] += b[i];
     return a;
   }
+
+  /**
```

Comment:
I don't understand this method (including method name)

Reply:
Never mind, I can replace it with standard add method.